main.js

```diff
@@ -0,0 +1,183 @@
+var destination = document.getElementById("destDrop");
+var makeMoveBtn = document.getElementById("submitMoveBtn");
+var message = document.getElementById("message");
+var gameBoardDisplay = document.getElementById("gameboard");
+
+// string messages
+var sameBayErr = "You must move the disc to a different bay than the one it occupies.";
+var emptyBayErr = "There is no item to move from selected bay. Choose another";
+var biggerDiscErr = "You cannot move a larger disc on top of a smaller one.";
+var successfulMove = "That move was successful, board is now: ";
+var gameWinner = "The game has been won!!!  Game board reset...";
+
+// game
+class Game {
+  constructor(defaultBoard, currentBoardState) {
+    this.defaultBoard = defaultBoard;
+    this.currentBoardState = currentBoardState;
+  }
+
+  getDefaultBoard(){
+    return this.defaultBoard;
+  }
+
+  setDefaultBoard(board){
+    board = defaultBoard;
+  }
+  getCurrentBoardState(){
+    return this.currentBoardState;
+  }
+  setCurrentBoard(board){
+    board = currentBoardState;
+  }
+
+}
+  // Game class getters and setters
+function getDefaultBoard(){
+    return this.defaultBoard;
+  }
+function setDefaultBoard(board){
+    board = defaultBoard;
+  }
+function getCurrentBoardState(){
+    return this.currentBoardState;
+  }
+function setCurrentBoard(board){
+    board = currentBoardState;
+  }
+
+  //function to move array items to different arrays
+function move(fromBoardIndex, toBoardIndex){
+    fromBoardIndex = start.value;
+    toBoardIndex = destination.value;
+
+    var bayToMoveFrom = getCurrentBoardState()[fromBoardIndex - 1];
+    var itemToMove = bayToMoveFrom[bayToMoveFrom.length-1];
+
+    bayStack = getCurrentBoardState()[toBoardIndex-1];
+    destTopItem = bayStack[bayStack.length-1];
+
+    switch(true){
+      case fromBoardIndex === toBoardIndex:
+        message.textContent = sameBayErr;
+        break;
+      case getCurrentBoardState()[fromBoardIndex-1].length === 0:
+        message.textContent = emptyBayErr;
+        break;
+      case itemToMove > destTopItem:
+        message.textContent = biggerDiscErr;
+        break;
+      default:
+        mover = getCurrentBoardState()[fromBoardIndex-1].pop();
+        setDefaultBoard(getCurrentBoardState()[toBoardIndex-1].push(mover));        
+        console.log("current board state: " + "\n" + mapBoard(getCurrentBoardState()));
+        gameBoardDisplay.innerHTML = mapBoard(getCurrentBoardState());        
+        console.log("current board state: " + "\n" + mapBoard(getCurrentBoardState()));
+        message.textContent = successfulMove;
+        console.log("current board state: " + "\n" + mapBoard(getCurrentBoardState()));
+        checkWinner(currentBoardState);
+    }
+  }
+
+  // function to map current board state
+function mapBoard(currentBoard){
+    var bay = 1;
+    return currentBoard.map(function (boardItem){
+      var line = `Bay ${bay++}: --- ${boardItem} <br/> \n`;

```

Comment:
String interpolation is an es6 feature - the rest of your code seems to be in a pre-es6 state. I recommend staying consistent.